src/meta/test/MetaClientTest.cpp

```diff
@@ -24,7 +26,7 @@ using nebula::cpp2::ValueType;
 using apache::thrift::FragileConstructor::FRAGILE;
 
 TEST(MetaClientTest, InterfacesTest) {
-    FLAGS_load_data_interval_second = 1;
+    FLAGS_load_data_interval_secs = 1;
```

Comment:
As for these tests, I think there are two ways to make them faster. First, we could have shared the same MockServer instead of launch one per test. Second, seconds is appropriate for a polling interval, but too long for unit testing, while I cannot come up with a better way. But you still could impove it by `usleep(FLAGS_load_data_interval_secs * 1000000 + 100000/*100ms*/)` to save 0.9s per sleep.

Of course, we could leave these work to the next PR.

Reply:
Sounds good